server.js

```diff
@@ -453,6 +465,56 @@ app.delete('/api/v1/users/:username', jsonParser, function(req, res) {
   });
 });
 
+app.post('/api/v1/places', jsonParser, function(req, res) {
+  auth.check(req, 'can_create_places', null, function(err) {
+    if (err) {
+      return serverUtils.error(err, req, res);
+    }
+    if (_.isEmpty(req.body)) {
+      return emptyJSONBodyError(req, res);
+    }
+    places.createPlace(req.body, function(err, body) {
+      if (err) {
+        return serverUtils.error(err, req, res);
+      }
+      res.json(body);
+    });
+  });
+});
+
+app.post('/api/v1/places/:id', jsonParser, function(req, res) {
+  auth.check(req, 'can_update_places', null, function(err) {
+    if (err) {
+      return serverUtils.error(err, req, res);
+    }
+    if (_.isEmpty(req.body)) {
+      return emptyJSONBodyError(req, res);
+    }
+    places.updatePlace(req.params.id, req.body, function(err, body) {
+      if (err) {
+        return serverUtils.error(err, req, res);
+      }
+      res.json(body);
+    });
+  });
+});
+
+app.post('/api/v1/contacts', jsonParser, function(req, res) {
+  auth.check(req, 'can_create_contacts', null, function(err) {
```

Comment:
I think this endpoint should be `/api/v1/people/` and the permission should be `can_create_people`.